script.js

```diff
@@ -0,0 +1,44 @@
+const loadMovieInfo = (name) => {
+    if (!name) { //This is runs when selected option is blank
+        document.querySelector('#movie_info').innerHTML = `<h3>To begin select a movie</h3>`;
+        document.querySelector('form').style.display = 'none';
+        return; //Prevents the rest of function from running
+    }
+    let filmObj = filmsObject[name];
+    let title = document.createElement('h3');
+    let year = document.createElement('p');
+    let description = document.createElement('p');
+    let filmArr = [title, year, description];
+    
+    title.innerText = filmObj.title;
+    year.innerText = filmObj.release_date;
+    description.innerText = filmObj.description;
+    filmArr = [title, year, description];
+    document.querySelector('form').style.display = 'block';
+    document.querySelector('#movie_info').innerText = '';
+    filmArr.forEach(item => document.querySelector('#movie_info').appendChild(item));
+}
+
+const addReview = () => {
+    let review = document.createElement('li');
+    review.innerHTML = `<strong>${document.querySelector('h3').innerText}:</strong> ${document.querySelector('input').value}`;
+    document.querySelector('input').value = '';
+    document.querySelector('ul').appendChild(review);
+}
+
+const createFilmOption = (name) => {
+    let option = document.createElement('option');
+    option.innerText = name;
+    console.log(option.inn);
+    
+    document.querySelector('select').appendChild(option);
```

Comment:
Since you are executing this function inside a loop multiple times and every time you get the select box from the DOM isn't this a bit inefficient? How could you instead grab the select once then loop over the film titles and append each one as an option

Reply:
could I have saved it as a variable and loop through it like that?